README.md

```diff
@@ -20,3 +20,17 @@ or
 ```
 
 Then navigate to `localhost:8181` in a browser.
+
+### Project Dashboard
+
+The Oppia Project Dashboard uses the web crypto API, which requires a TLS connection.
+To start an HTTPS simple server with a generated self-signed certificate, execute:
+
+```shell
+$ python3 start.py
```

Comment:
Suggest mentioning somewhere in this section that only Python 3 is supported when accessing the project dashboard (just to be explicit since above we have both Python 2 & 3 instructions).

Reply:
I think since Python 2 has been deprecated for over a year it might be okay to remove python2 instructions.  Thoughts?

Reply:
We're planning to keep the Python 2 instruction since the Oppia web backend still uses it.